Orchestrator/README.md

```diff
@@ -138,9 +138,20 @@ const myDialog = new AdaptiveDialog('myDialog').configure({
 
 ## Composer Integration
 
-\<TBD: This section is FYI in preparation for upcoming Composer functionality. It will be updated once ready.>
+Orchestrator has been can be used as recognizer in [Bot Framework Composer][19]. To specify Orchestrator as a dialog recognizer:
 
-Once the feature flag is enabled in Composer, it is possible to specify Orchestrator as a recognizer. For the most basic intent recognition cases, simply specify Orchestrator as the recognizer, and fill in the language data as you would for LUIS. For more advanced scenarios, such as dispatch orchestration, follow the steps above to import and tune up routing quality. 
+1. Enable Orchestrator feature in Composer's Application Settings
+2. In desired dialog select Orchestrator the recognizer dropdown menu
```

Comment:
select Orchestrator in the recognizer dropdown menu

Reply:
I thought it'd be good to indicate that it is per dialog.

Reply:
Ok. it works this way as well...